### ISSUE TITLE
[A11y] General - Hamburgers and High Contrast

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1986]
### Operating System

Windows

### Browser

Edge

### Affected page

Homepage

### Admin user?

Yes

### Issue Description

Only just thought to check the combination of high contrast / zoom (doing a "mobile" test). Hamburger menus disappear:

![image](https://user-images.githubusercontent.com/2157974/130815407-1c4a178c-3a79-4c1e-8466-72040dc5d26d.png)

I don't know if this precise combination is a WCAG failure, but I suspect there are users who have high zoom levels and high contrast turned on, so it seems like it could be.

### Suggested Fix

Consider an SVG for these menus (though I can't find one myself) or "real" lines rather than having CSS-generated content. May not be easy to reinvent this particular wheel, though.

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

Zoom or reduce the window until "hamburger menus" appear, turn on high contrast, see if they're still visible.

**Comment by azle** [comment]
Updated 12/30/2024 FIXED
